apps/frontend/src/lib/stores/locale.ts

```diff
@@ -0,0 +1,7 @@
+import { getLocale, locales } from '$lib/paraglide/runtime.js';
+import { writable } from 'svelte/store';
```

Comment:
Should use Svelte 5 runes, not stores.

Ref: https://svelte.dev/docs/svelte/what-are-runes, https://svelte.dev/blog/runes

Reply:
Note that Paraglide JS already has `getLocale`, so I'm not sure we need a store at all.